helm/trivy/templates/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "trivy.fullname" . }}
+  labels:
+{{ include "trivy.labels" . | indent 4 }}
+spec:
```

Comment:
By default this service has `spec.sessionAffinity: None`, which might cause inconsistency errors when we scale up StatefulSet replicas. I was actually able to see such errors in my env after scaling to 3 replicas.

Basically we have to make sure that all requests related to scanning a particular image are routed to the same pod, which has its own file system cache (PVC -> PV).

The following sketch diagram depicts the problem that I'm trying to describe.

![trivy-server-session-affinity](https://user-images.githubusercontent.com/1322923/100794800-5edae880-341e-11eb-80ad-6d915ced84cf.png)

One way to fix it is to change `spec.sessionAffinity: None` to `spec.sessionAffinity: ClientIP`, but we have to double check how does that impact scalability. /cc @knqyf263

Reply:
Added

Reply:
I've supported Redis and it will address this issue.
https://github.com/aquasecurity/fanal/pull/143

Should we wait for the Redis support? Or, can we merge this PR once and replace PV with Redis in separate PR? What do you think, @danielpacak?

Reply:
It will be done in the subsequent PR.